src/types/rooms.ts

```diff
@@ -3,6 +3,7 @@ import { SoundConfigReference } from "./sounds";
 export enum RoomType {
   unclickable = "UNCLICKABLE",
   mapFrame = "MAPFRAME",
+  modalFrame = "MODALFRAME",
 }
```

Comment:
> > I had to make a small change on rooms.ts to avoid the error `Property 'modalFrame' does not exist on type 'typeof RoomType'` in `RoomModal.tsx` which happened when I tried to run the `env/github` code locally
> 
> @malaikaaiyar See this thread: https://doesitsparkle.slack.com/archives/C01ER4M61TL/p1624580015339300
> 
> _Originally posted by @0xdevalias in https://github.com/sparkletown/sparkle/issues/1654#issuecomment-868152196_

Reply:
So should I leave this change or is it being fixed in a different PR?

Reply:
It needs to made in a separate PR to get it fixed sooner. I’ve intended to do it all day but keep getting sidetracked.

I was trying to figure out how it even came to be an issue in the first place. Though can’t figure that out still tbh. I’m thinking maybe some weird merge conflicts happened and some changes were lost during that.. but not sure

Reply:
It’s fine to leave here but. Worst case if I fix it before you (or someone else does) then it’ll just see it as the same change/give you a conflict/sinilar